resources/execute.rb

```diff
@@ -0,0 +1,26 @@
+default_action(:run)
```

Comment:
NIT: Remove parentheses here, for property and action to be consistent with other resources